pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -262,6 +263,11 @@ func TestToKubeContainerStatusWithResources(t *testing.T) {
 						MemoryLimitInBytes: 524288000,
 						OomScoreAdj:        -998,
 					},
+					Windows: &runtimeapi.WindowsContainerResources{
```

Comment:
In practice, we won't have both `LinuxContainerResources` and `WindowsContainerResources` would we? Perhaps we should populate the input.Resources based on the platform the test is executing on and select either LCR or WCR?